.github/workflows/release-new-version.yml

```diff
@@ -0,0 +1,94 @@
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: 11
+          distribution: temurin
+          server-id: ossrh
+          server-username: OSSRH_TOKEN_USERNAME
+          server-password: OSSRH_TOKEN_PASSWORD
+          gpg-private-key: ${{ secrets.JAVA_NATIVE_PGP_KEY }}
+          gpg-passphrase: PGP_KEY_PASSPHRASE
+      - name: Set release version if provided as input
+        if: github.event.inputs.releaseVersion != ''
+        run: echo "VERSIONS=-DreleaseVersion=${{ github.event.inputs.releaseVersion }}" >> $GITHUB_ENV
+      - name: Publish artifacts to Maven Central
+        run: mvn -B release:prepare release:perform -P package,maven-central-release ${VERSIONS}
+        env:
+          OSSRH_TOKEN_USERNAME: ${{ secrets.OSSRH_TOKEN_USERNAME }}
+          OSSRH_TOKEN_PASSWORD: ${{ secrets.OSSRH_TOKEN_PASSWORD }}
+          PGP_KEY_PASSPHRASE: ${{ secrets.JAVA_NATIVE_PGP_KEY_PASSPHRASE }}
+      - run: echo "releaseTag=$(git tag --points-at HEAD^)" >> $GITHUB_ENV
+      - run: echo "releaseVersion=$(echo ${{ env.releaseTag }} | cut -c2-)" >> $GITHUB_ENV
+      - name: Push changes back to repo
+        run: git push && git push --tags --force
+    outputs:
+      releaseTag: ${{ env.releaseTag }}
+      releaseVersion: ${{ env.releaseVersion }}
+
+  github-release:
+    runs-on: ubuntu-latest
+    needs: maven-central-release
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ needs.maven-central-release.outputs.releaseTag }}
+      - run: git config --global user.email "${GITHUB_BOT_EMAIL}"
+      - run: git config --global user.name "${GITHUB_BOT_NAME}"
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: 11
+          distribution: temurin
+      - name: Build jar with dependencies included
+        run: mvn -B install -Ppackage,jar-with-dependencies
+      - name: Publish as release
+        uses: ncipollo/release-action@v1
+        with:
+          name: ${{ needs.maven-central-release.outputs.releaseVersion }}
+          tag: ${{ needs.maven-central-release.outputs.releaseTag }}
+          artifacts: target/*.jar
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+  update-version-references:
+    runs-on: ubuntu-latest
+    needs: maven-central-release
+    env:
+      ver: ${{ needs.maven-central-release.outputs.releaseVersion }}
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+            git config --global user.email "${GITHUB_BOT_EMAIL}"
+            git config --global user.name "${GITHUB_BOT_NAME}"
+      - run: |
+            sed -i "/<artifactId>jssc<\/artifactId>/{n;s/<version>.*<\/version>/<version>${ver}<\/version>/}" README.md
+            sed -i "s/io.github.java-native:jssc:.*\"/io.github.java-native:jssc:${ver}\"/" README.md
+            sed -i "s/io.github.java-native:jssc:.*'/io.github.java-native:jssc:${ver}'/" README.md
```

Comment:
```suggestion
            sed -i'.bak' -e "s/java-native:jssc:[A-Za-z0-9.-]*/java-native:jssc:${ver}\/g" README.md
```

Reasoning:
* `-i'.bak' -e` makes the call work on both macOS and GNU sed
* `/g` replaces all, removes the redundant call
* `[A-Za-z0-9.-]` matches any `.patch` value, but not the single or double-quote, e.g. `2.9.`**`5-RC1`**

Some additional notes:
* I would recommend a global search `/g` is always used with sed, there's no reason not to.
* Note, if you wish, this call can be repeated on `https://github.com/java-native/jssc.wiki` repo.

To test the `jssc.wiki` repo locally:

```bash
git clone https://github.com/java-native/jssc.wiki && cd jssc.wiki
sed -i'.bak' -e "s/java-native:jssc:[A-Za-z0-9.-]*/java-native:jssc:9.9.9/g" Add-Dependency.md
git diff
cd .. && rm -rf jssc.wiki
```

Which should preview as:

```diff
diff --git a/Add-Dependency.md b/Add-Dependency.md
index 29a6916..6ee524d 100644
--- a/Add-Dependency.md
+++ b/Add-Dependency.md
@@ -16,7 +16,7 @@ repositories {
     mavenCentral()
 }
 dependencies {
-    implementation("io.github.java-native:jssc:2.9.4")
+    implementation("io.github.java-native:jssc:9.9.9")
 }
 
@@ -26,6 +26,6 @@ repositories {
     mavenCentral()
 }
 dependencies {
-    implementation 'io.github.java-native:jssc:2.9.4'
+    implementation 'io.github.java-native:jssc:9.9.9'
 }
```

Reply:
My original MR included wiki update, but then it started to bother me - that's two different repositories after all, and especially, the wiki, by definition is alive. I wouldn't like the build to fail on the last stage - after all other steps are completed, just because we decide to rename a file or change formatting or some such. 
A different idea may be to keep readme as a wiki page? If including content from the main repo is not possible (and it seems it is not at this point) then perhaps dropping entire README.MD as a front page of wiki instead of doing partial content redactions would do.

Reply:
I fundamentally dislike it when projects include constantly changing documentation in the `README.md`as stated here https://github.com/java-native/jssc/pull/68#issuecomment-667295172.  I'd be happy to remove this information to although I don't think it addresses the problem of the wiki becoming out of date on each release.

> build to fail on the last stage - after all other steps are completed, just because we decide to rename a file or change formatting

We can do a global replace on all wiki markdown files then.  I understand the reservation however if we're to automate the manual processes, this seems like a good candidate, no?

Reply:
Added action for wiki update